actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -171,6 +171,7 @@ def tag_options(options, escape = true)
         end
 
         def prefix_tag_option(prefix, key, value, escape)
+          return "" if value.nil?
```

Comment:
We could move this check to like 157 and avoid calling this method and build the string.